Extension/src/LanguageServer/client.ts

```diff
@@ -1395,6 +1396,20 @@ export class DefaultClient implements Client {
         return this.queueTask(() => Promise.resolve(this.configuration.CurrentConfiguration.name));
     }
 
+    public setCurrentConfigName(configurationName: string): Thenable<void> {
+        return this.queueTask(() => new Promise((resolve, reject) => {
+            let configurations : configs.Configuration[] = this.configuration.Configurations || [];
+            let configurationIndex : number = configurations.findIndex((config) => config.name === configurationName);
+
+            if (configurationIndex !== -1) {
+                this.configuration.select(configurationIndex);
+                resolve();
+            } else {
+                reject(new Error(`requested configuration name is not found: ${configurationName}`));
```

Comment:
To get this string localized, could you wrap this string in a call to localize() ?  The first string passed to localize() just needs to be a unique identifier (maybe "config.not.found"?).  With localize(), you'll need to use "{0}" and pass in configurationName as an additional argument, instead of using string interpolation.

Reply:
ack. will do via `localize` wrapper.